xm-commons-logging-configurable/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'io.spring.dependency-management'
+dependencyManagement {
+    imports {
+        mavenBom("org.springframework.boot:spring-boot-dependencies:$versions.springBoot")
+        mavenBom "org.springframework.cloud:spring-cloud-dependencies:${versions.springCloud}"
+    }
+
+}
+
+dependencies {
+    compile project(':xm-commons-config')
+    compile project(':xm-commons-lep')
+    compile "org.springframework:spring-context"
+
+    compile "org.slf4j:slf4j-api"
+    compile "ch.qos.logback:logback-classic"
+    compile "org.aspectj:aspectjweaver"
+
+    compile "org.apache.commons:commons-lang3:${versions.commonsLang3}"
+    compile "commons-codec:commons-codec:${versions.commonsCodec}"
+    compile ("net.logstash.logback:logstash-logback-encoder:${versions.logstashLogbackEncoder}") {
+        exclude(group: 'ch.qos.logback')
+    }
+
+    compile 'org.springframework.cloud:spring-cloud-commons'
```

Comment:
do we need spring Cloud deps for this module?

Reply:
deleted